Makefile

```diff
@@ -69,62 +69,66 @@ PACKAGE_NAME?=github.com/kelseyhightower/confd
 # All go files.
 SRC_FILES:=$(shell find . -name '*.go' -not -path "./vendor/*" )
 
-DOCKER_GO_BUILD := mkdir -p .go-pkg-cache && \
-                   docker run --rm \
-                              --net=host \
-                              $(EXTRA_DOCKER_ARGS) \
-                              -e LOCAL_USER_ID=$(LOCAL_USER_ID) \
-                              -e GOARCH=$(ARCH) \
-                              -v $(HOME)/.glide:/home/user/.glide:rw \
-                              -v ${CURDIR}:/go/src/$(PACKAGE_NAME):rw \
-                              -v ${CURDIR}/.go-pkg-cache:/go/pkg:rw \
-                              -w /go/src/$(PACKAGE_NAME) \
-                              $(CALICO_BUILD)
+EXTRA_DOCKER_ARGS	:= -e GO111MODULE=on
+
+ifdef GOPATH
+	EXTRA_DOCKER_ARGS += -v $(GOPATH)/pkg/mod:/go/pkg/mod:rw
+endif
+
+DOCKER_RUN := mkdir -p .go-pkg-cache && \
+	docker run --rm \
+		--net=host \
+		$(EXTRA_DOCKER_ARGS) \
+		-e LOCAL_USER_ID=$(LOCAL_USER_ID) \
+		-e GOCACHE=/go-cache \
+		-e GOARCH=$(ARCH) \
+		-e GOPATH=/go \
+		-v $(CURDIR):/go/src/$(PACKAGE_NAME):rw \
+		-v $(CURDIR)/.go-pkg-cache:/go-cache:rw \
+		-w /go/src/$(PACKAGE_NAME)
 
 .PHONY: clean
 clean:
+	rm -rf vendor
 	rm -rf bin/*
 	rm -rf tests/logs
 
 ###############################################################################
 # Building the binary
 ###############################################################################
-build: bin/confd
+.PHONY: remote-deps
+remote-deps:
+	$(DOCKER_RUN) $(CALICO_BUILD) sh -c '\
+	go list all; \
+	mkdir -p vendor/golang.org/x/crypto; \
+	cp -r `go list -m -f "{{.Dir}}" golang.org/x/crypto`/openpgp vendor/golang.org/x/crypto/openpgp; \
+	chmod -R +w vendor/golang.org/x/crypto'
+
```

Comment:
I don't think these remote deps are needed.  I removed this section and things seem to build fine --- why did you add them?

Reply:
Probably leftover from another issue, let me commit the removal...

Reply:
Fixed